Tickets/preops 1316 (#11)
lsst/rtn-011: intro.tex

The Operations team is tracking the progress of the commissioning activities as they relate to \es opportunities to ensure that the community has prompt access to science-ready data products while the survey begins its relentless coverage of the sky leading to DR1.
We broadly envisage two possible scenarios emerging from the commissioning phase of the construction project: 

\begin{itemize}
\item \textbf{Scenario A}:
The full commissioning plan comprising system optimization and science validation is successfully executed as planned. 
The Operations team will then carry out an Operations Readiness Review (ORR) to effectively conduct a “full dress rehearsal” of science operations and demonstrate the readiness of the Operations team to execute the 10-year survey. 
Data collected during commissioning is reprocessed to produce DP2, which will be released 6 months follwoing the completion of the SV Surveys, see S~\ref{tab:dr-one-products}.

\item \textbf{Scenario B}:
On-sky time in commisisoning is reduced further resulting in the science validation surveys not being completed prior to the construction Operations Readiness Review (ORR). 

> In this scenario, the Operations team would spend up to 3 months prior to commencing the 10-year survey completing any remaining Science Validation Survey observations and releasing DP2 with the exact same data products as planned but 

Reviewer: ```suggestion
In this scenario, the Operations team would spend up to 3 months prior to commencing the 10-year LSST survey carrying out any remaining Science Validation Survey observations and releasing DP2 with the same data products as planned in Scenario A.
```

\end{itemize}

> An key point to note is that the contents of DP2 will be unchanged irrespective of which scenario materializes.  

Reviewer: ```suggestion
A key point to note is that the contents of DP2 will be the same, irrespective of which scenario materializes.  
```

Only the timing of the release of DP2 and the start of the 10-year survey are difrerent between the two scenarios. 
In both scenarios it is assumed that the Rubin Construction project delivers an integrated system that can capture, transfer and process science-grade data at the time Operations begins.
Both scenarios will include alert generation of some type, with the major distinction being the relative availability of templates in time, sky position, and filter.

> These scenarios presented are current at the time of writing, but as this document is "living", we expect the plans to mature as we approach full survey operations and the extant commissioning program emerges and is executed.

Reviewer: ```suggestion
These two scenarios presented are current as of October 2022, but since this document is ``living,'' we expect the plans to mature as we approach full survey operations and the commissioning program emerges and is executed.
```
